client_test.go

```diff
@@ -51,35 +51,6 @@ func TestClient_New(t *testing.T) {
 	}
 }
 
-// TestClient_InstantiationCreatesRepositoriesPath ensures that instantiating the
-// client has the side-effect of ensuring that the repositories path exists
-// on-disk, and also confirms that the XDG_CONFIG_HOME environment variable is
-// respected when calculating this home path.
-func TestClient_InstantiationCreatesRepositoriesPath(t *testing.T) {
```

Comment:
This test was reloacted to both config (testing that CreatePaths works as expected) and the cli (ensuring add repositories works requires the path exist)